cmd/security-agent/app/flare.go

```diff
@@ -81,6 +81,9 @@ func requestFlare(caseID string) error {
 	urlstr := fmt.Sprintf("https://localhost:%v/agent/flare", config.Datadog.GetInt("security_agent.cmd_port"))
 
 	logFile := config.Datadog.GetString("log_file")
+	if logFile == "" {
+		logFile = common.DefaultSecuriyAgentLogFile
```

Comment:
If no log file is specified in config, the `security-agent` will now use a different default log file from the core agent.
But if a log file is specified in the config, then the `security-agent` and the core agent will both use it.
Shouldn’t we have another config item for the `security-agent` log file ?

Reply:
done